pkg/controller/kabaneroplatform/featured_stacks.go

```diff
@@ -72,10 +81,13 @@ func reconcileFeaturedStacks(ctx context.Context, k *kabanerov1alpha2.Kabanero,
 			for j, stackVersion := range stackResource.Spec.Versions {
 				if stackVersion.Version == stack.Version {
 					foundVersion = true
-					stackVersion.Pipelines = stack.Pipelines
-					stackVersion.SkipCertVerification = stack.SkipCertVerification
-					stackVersion.Images = stack.Images
-					stackResource.Spec.Versions[j] = stackVersion
+					// Per the new defintion of active, do not update any existing stacks with matching names and versions.
+					if !(alreadyDeployed && stackVersion.DesiredState == "active") {
```

Comment:
I'm wondering if this should be `len(stackVersion.DesiredState) > 0` instead of `stackVersion.DesiredState == "active"`.  If someone sets this to "inactive" I think we'll be changing the pipelines and images in the Stack CR.  The user may not be expecting that, since they set `desiredState` to a non-null value.